docker-compose.yml

```diff
@@ -1,15 +1,15 @@
 version: "2"
 services:
   nginx:
-    image: dopplerdev/doppler-for-shopify-nginx
+    image: dopplerdev/doppler-for-shopify-nginx-qa
     ports:
       - 443:443
     links:
       - app
     networks:
       - app
     volumes:
-      - ${SSL_CERT_PATH}:/etc/nginx/cert # Define this as an environment variable in Docker in host
+      - /etc/ssl/fromdoppler.com:/etc/nginx/cert
```

Comment:
Did you choose to hardcode it? or it is only for your local environment?

Reply:
Nevermind, I'm going to revert this change.